frontend/coprs_frontend/coprs/templates/coprs/detail/_builds_forms.html

```diff
@@ -161,15 +161,12 @@ <h3 class="panel-title">{{ counter('instructions') }}. Select chroots and other
        )
     }}
 
-    {{ render_field(form.pypi_package_name, placeholder="Package name in the Python Package Index.") }}
+    {{ render_field(form.pypi_package_name, placeholder="{{ common_descriptions.PYPI_PACKAGE_NAME.description }}.") }}
     {{ render_field(form.pypi_package_version, placeholder="Optional - Version of the package PyPI") }}
 
     {{ render_field(
          form.spec_generator,
-         info="Tool for generating specfile from a PyPI package. The options "
-               "are full-featured <strong>pyp2rpm</strong> with cross "
-               "distribution support, and <strong>pyp2spec</strong> that is "
-               "being actively developed and considered to be the future."
+         info="{{ common_descriptions.SPEC_GENERATOR.description }}"
```

Comment:
Here we are losing formatting.

Reply:
Also note there are some form field metdata arguments we could perhaps use: https://github.com/fedora-copr/copr/blob/0b7d4587e5ddaf3185684b5c83aef7f18ce4fc97/frontend/coprs_frontend/coprs/forms.py#L581-L584

Reply:
> Here we are losing formatting.

that's true... perhaps this one is worth not unifying